Servatrice Debug Build throws build error's

When building servatrice with -DCMAKE_BUILD_TYPE=Debug the following errors are thrown.
Most likely related to the websocket PR (#1991).

```
Cockatrice/servatrice/src/servatrice.cpp:106:1: error: unused parameter ‘_numberPools’ [-Werror=unused-parameter]
 Servatrice_WebsocketGameServer::Servatrice_WebsocketGameServer(Servatrice *_server, int _numberPools, const QSqlDatabase &_sqlDatabase, QObject *parent)
 ^
cc1plus: all warnings being treated as errors
make[2]: *** [servatrice/CMakeFiles/servatrice.dir/src/servatrice.cpp.o] Error 1
make[1]: *** [servatrice/CMakeFiles/servatrice.dir/all] Error 2
```

--- a/servatrice/src/servatrice.cpp
+++ b/servatrice/src/servatrice.cpp
@@ -103,7 +103,7 @@ Servatrice_ConnectionPool *Servatrice_GameServer::findLeastUsedConnectionPool()
 #if QT_VERSION > 0x050300
 #define WEBSOCKET_POOL_NUMBER 999
 
-Servatrice_WebsocketGameServer::Servatrice_WebsocketGameServer(Servatrice *_server, int _numberPools, const QSqlDatabase &_sqlDatabase, QObject *parent)
+Servatrice_WebsocketGameServer::Servatrice_WebsocketGameServer(Servatrice *_server, int /* _numberPools */, const QSqlDatabase &_sqlDatabase, QObject *parent)
     : QWebSocketServer("Servatrice", QWebSocketServer::NonSecureMode, parent),
       server(_server)
 {

